script/scriptinterface.cpp

```diff
@@ -71,6 +71,36 @@ void ScriptInterface::exportMetrics() {
   log("Metrics exported to application directory.");
 }
 
+QVariant ScriptInterface::getMetrics(QString name, bool history) {
+  if (history) {
```

Comment:
I believe this whole implementation could be more compact written as:
```
for (const auto& c : sim.getSystem()->getCounts()) {
  if (c->_name == name) {
    return history ? c->_history : c->_value;
  }
}
for (const auto& m : sim.getSystem()->getMeasures()) {
  if (m->_name == name) {
    return history ? c->_history : c->_value;
  }
}
log("no metrics with given name exist", true);
return QVariant();
```
It's functionally equivalent to what you've written, but less repetitive and more compact.